spray-http/src/main/scala/spray/http/CacheDirective.scala

```diff
@@ -45,7 +50,6 @@ object CacheDirectives {
   case object `no-cache` extends SingletonValueRenderable with RequestDirective with ResponseDirective
   case object `no-store` extends SingletonValueRenderable with RequestDirective with ResponseDirective
   case object `no-transform` extends SingletonValueRenderable with RequestDirective with ResponseDirective
-
   case class `max-age`(deltaSeconds: Long) extends RequestDirective with ResponseDirective with ValueRenderable {
```

Comment:
This one would also be eligible for using `ResponseDirectiveWithDeltaSeconds` as 

```scala
case class `max-age`(deltaSeconds: Long) extends ResponseDirectiveWithDeltaSeconds with RequestDirective
```